### ISSUE TITLE
Question about Fine-tune

### ISSUE THREAD

**Issue by XuShoweR** [TuSimple/simpledet#146]
If I directly use  python detection_train.py --config config/model_config.py  and experiments/model/model_0018.params exists ,will it load by the model?Or i need to change begin_epoch to 18. and my class_num is 1 + 10 , i need to delete the head in the params or is there another method to use.

**Comment by RogerChern** [comment]
Your understanding is correct.
https://github.com/TuSimple/simpledet/blob/6f826134cf699223caee258f91d824706eab6bb9/detection_train.py#L127-L128

You should change the `begin_epoch`, `num_class` and delete the head params from `arg_params`.

Maybe we can add a more detailed guide for finetuning models from existing checkpoint.

**Comment by XuShoweR** [comment]
> You should change the `begin_epoch`, `num_class` and delete the head params from `arg_params`.
Thanks for your reply, it helps a lot. And I observe the params ,you mean i need to delete "bbox_cls_logit_bias", "bbox_cls_logit_weight"？

**Comment by XuShoweR** [comment]
> Your understanding is correct.
> https://github.com/TuSimple/simpledet/blob/6f826134cf699223caee258f91d824706eab6bb9/detection_train.py#L127-L128
> 
> You should change the `begin_epoch`, `num_class` and delete the head params from `arg_params`.
> 
> Maybe we can add a more detailed guide for finetuning models from existing checkpoint.

Thanks for your reply, it helps a lot. And I observe the params ,you mean i need to delete "bbox_cls_logit_bias", "bbox_cls_logit_weight"？

**Comment by RogerChern** [comment]
Hi, @XuShoweR. 
1. Yes, delete the weight and bias of `bbox_cls_logit`.
2. It should be fine for declaring more classes than there actually is. Since we obtain the final classification result from a softmax activation, I think the unseen classes should have large negative biases as they are always panelized by the cross-entropy loss.

**Comment by XuShoweR** [comment]
> Hi, @XuShoweR.
> 
> 1. Yes, delete the weight and bias of `bbox_cls_logit`.
> 2. It should be fine for declaring more classes than there actually is. Since we obtain the final classification result from a softmax activation, I think the unseen classes should have large negative biases as they are always panelized by the cross-entropy loss.

According to your guidance, training has begun .Thank you very much for your timely and patient reply.Thanks！
